app-template/src/example-configuration/routesConfig.ts

```diff
@@ -0,0 +1,117 @@
+import { GenericRoute } from '../types/portal-config'
```

Comment:
tests could be written to go against real portals

Reply:
but maybe they shouldn't be (can put union of all portal code into example and test that).